erp-server/src/main/java/com/ttt/erp/service/UserAccountService.java

```diff
@@ -41,25 +41,23 @@ public UserAccount userFromPrincipal(Principal principal) {
         return Optional.ofNullable(this.repository.findByUsername(username).getSignature());
     }
 
-    public Optional<UserAccount> createUser(Long userAccountId, UserAccount user) {
+    public Optional<UserAccount> createUser(UserAccount user) {
```

Comment:
This also gets called on admin user creates. Was the function signature changed for that controller?

Reply:
No, I didn't realize the admin user creation used this too. I'll check on it.

Reply:
Wait, actually -- where do I need to look for that?

Reply:
I don't think we can do this. the userAccountId param is supposed to be for the actingUser (the one submitting the request on the client). By removing it we'll never log any user creations by the admin user.

Reply:
Yeah you were right. It needed to be put back in place and handled differently.